src/doc.d

```diff
@@ -2563,7 +2566,7 @@ extern (C++) void highlightCode(Scope* sc, Dsymbols* a, OutBuffer* buf, size_t o
                 parametersBuf.writeByte(')');
 
                 const templateParams = parametersBuf.peekString();
-                const templateParamsLen = strlen(templateParams);
+                const templateParamsLen = parametersBuf.peekSlice().length;
```

Comment:
I thought about it, but there weren't enough use cases. Inlining peekSlice() should take care of any efficiency concerns.